src/style.css

```diff
@@ -28,3 +28,12 @@
   position: absolute;
   z-index: 1000;
 }
+
+.leaflet-top {
+  bottom: 0;
+  }
+  
+  .leaflet-top .leaflet-control-zoom {
+  top:90%;
+  transform: translateY(-50%);
+  }
```

Comment:
This styles will break map layout. Remove them.  
In `src/the-map.tsx` add this before `TheMap` declaration

```js
import { divIcon, Control } from 'leaflet'

// ...
Control.Zoom.prototype.options.position = 'bottomleft'

export function TheMap({ selectedRoutes, showUserLocation }: Props) {
  // ...
```